src/questions/derivatives.yaml

```diff
@@ -0,0 +1,948 @@
+- uuid: e0b6dff3-946e-4e8d-9f23-8728fbb37c2a
+  type: SingleChoice
+  content: What determines the value of derivatives?
+  hint: NoHint
+  explanation: Derivatives derive their value from the underlying asset they are representing.
+  answerKeys:
+    - B
+  subTopics:
+    - "introduction-to-derivatives"
+  difficultyLevel: "Low"
+  choices:
+    - content: Demand and Supply of derivative
+      key: A
+    - content: An asset from which it derives its value
+      key: B
+    - content: Fixed Price set by the owner
+      key: C
+    - content: Standardised Prices are set by the Government.
+      key: D
+- uuid: ad7116fc-e76e-4e5b-b98f-55cea5c7e9e3
+  type: SingleChoice
+  content: Financial Derivatives consist of-
+  hint: NoHint
+  explanation: Financial Derivatives consist of Forwards, Futures, Options, Swaps
+  answerKeys:
+    - C
+  subTopics:
+    - "introduction-to-derivatives"
+  difficultyLevel: "Low"
+  choices:
+    - content: Stocks
+      key: A
+    - content: Bonds
+      key: B
+    - content: Futures
+      key: C
+    - content: None of the above 
+      key: D
+- uuid: 16f39eef-f4d9-47dc-9c9d-2f3b34dea7f9
+  type: SingleChoice
+  content: Porfolio Hedging is done by bank manager to- 
+  hint: NoHint
+  explanation: Hedging is done primarily for reducing the interest rate risk.
+  answerKeys:
+    - A
+  subTopics:
+    - "introduction-to-derivatives"
+  difficultyLevel: "Medium"
+  choices:
+    - content: Reduce interest rate risk.
+      key: A
+    - content: Increase reinvestment risk
+      key: B
+    - content: Increase exchange rate risk.
+      key: C
+    - content: Increase the probability of gains.
+      key: D
+- uuid: b59e8a13-3f22-4b14-9f27-c9050ce63a62
+  type: SingleChoice
+  content: Payoffs in financial derivatives are connected to-
```

Comment:
I didn't get this question

Reply:
In this question I am trying to ask from the user that the financial derivatives are payed off in which asset

Reply:
We can just take this off